projects/ui/src/shim.cds-core.scss

```diff
@@ -486,6 +486,20 @@
   --clr-forms-range-progress-fill-color: var(--cds-alias-status-info); // var(--clr-color-action-600);
   --clr-forms-range-track-color: var(--cds-alias-object-border-color); // var(--clr-color-neutral-200);
 
+  // Password
+  --clr-forms-password-input-height: var(--cds-global-space-9); // input height 24px
+  --clr-forms-password-padding-bottom: var(--cds-global-space-5); // password padding
+  --clr-forms-password-layout-space: var(--cds-global-space-3); // layout space
+  --clr-forms-password-eye-icon-min-height: var(--cds-global-space-9); // show/hide icon min-height
+  --clr-forms-password-eye-icon-width: var(--cds-global-space-9); // show/hide icon width
+
+  --clr-forms-password-typography-color: var(--cds-alias-typography-color-400); // not present for the moment
+
+  // TODO: Make sure this is what we need,
+  //  in spec it is --cds-alias-interaction-color, which is gray, hence it's put as fallback here
+  --clr-forms-password-eye-icon-color: var(--cds-global-color-blue-700, --cds-alias-object-interaction-color);
+  --clr-forms-password-disabled-eye-icon-color: var(--cds-alias-status-disabled);
+  --clr-forms-password-disabled-typography-color: var(--cds-alias-typography-disabled, var(--cds-alias-typography-color-400));
```

Comment:
Fallbacks from typography should be mapped to their globals 
`var(--cds-alias-typography-disabled, var(--cds-global-typography-disabled))`

The `-alias-typography` values don't exist yet, but they will directly map to the `-global-typography` values.

Reply:
I see, the global typography disabled hasn't landed yet in Core. Disregard this comment.